docs/docs/react-test-renderer.md

```diff
@@ -0,0 +1,213 @@
+---
+id: react-test-renderer
+title: React Test Renderer
+permalink: docs/react-test-renderer.html
+layout: docs
+category: Reference
+---
+
+**Importing**
+
+```javascript
+import ReactTestRenderer from 'react-test-renderer'; // ES6
+const ReactTestRenderer = require('react-test-renderer'); // ES5 with npm
+```
+
+## Overview
+
+This package provides an experimental React renderer that can be used to render React components to pure JavaScript objects, without depending on the DOM or a native mobile environment.
+
+Essentially, this package makes it easy to grab a snapshot of the "DOM tree" rendered by a React DOM or React Native component without using a browser or jsdom.
```

Comment:
"DOM tree" is a bit iffy when said about React Native. Maybe "a snapshot of the platform view hierarchy (similar to a DOM tree)"?